src/components/common/sidebar/app-sidebar.tsx

```
@@ -1,18 +1,25 @@
   Briefcase,
   Calendar,
+  ChevronDown,
+  ChevronRight,
   CreditCard,
+  Database,
   FileDown,
   FileText,
   Heart,
   Home,
   LogOut,
   Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)